frontend/src/static/css/style.scss

```diff
@@ -176,8 +175,10 @@ header{
             padding-right:0.25rem;
         }
 
-        &--tooltip{
+        &--code-view-button, &--repo-url-button{
```

Comment:
if the style is the same, we will try to use the same modifier.

in this case, both should be `--button`